test/fixtures/runtime/package.json

```diff
@@ -2,6 +2,6 @@
   "name": "runtime-1",
   "version": "1.0.42",
   "dependencies": {
-    "platformatic": "^1.24.0"
+    "platformatic": "1.25.0"
```

Comment:
```suggestion
    "platformatic": "^1.25.0"
```

Reply:
The fixed version on the fixture is on purpose, to test the fact that the application list calculate correctly  if there is a new platformatic version.